### PR TITLE
DOC-7522 Add XDCR-Inter-Sync Gateway note

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-7522

### CODE REVIEW

File: modules/ROOT/pages/_partials/_page-index.adoc [hunk 1]
```diff
@@ -232,8 +232,8 @@ endif::xref-pfx-sgw[]
 
 // [Inter-sync-gateway Replication]
 :xref-sgw-pg-icr-sgreplicate-init: {xref-sgw-pfx-icr-sgreplicate}#initialization[[Inter-Sync Gateway Replication]]
-:xref-sgw-pg-icr-sgreplicate-sgr1: {xref-pfx-sgw}{sgw-pg-icr-sgreplicate-sgr1}[Inter-Sync Gateway Replication vn:1]
-:xref-sgw-pg-icr-sgreplicate-sgr2: {xref-sgw-pfx-icr-sgreplicate}[Inter-sync-gateway Replication]
+:xref-sgw-pg-icr-sgreplicate-sgr1: {xref-pfx-sgw}{sgw-pg-icr-sgreplicate-sgr1}[SG Replicate]
+:xref-sgw-pg-icr-sgreplicate-sgr2: {xref-sgw-pfx-icr-sgreplicate}[Inter-Sync Gateway Replication]
```
Comment (rajagp) [inline modules/ROOT/pages/_partials/_page-index.adoc:236]:
```suggestion
:xref-sgw-pg-icr-sgreplicate-sgr2: {xref-sgw-pfx-icr-sgreplicate}[inter-Sync Gateway Replication with Sync Gateway 2.8+]
```

File: modules/ROOT/pages/_partials/block-caveats.adoc [hunk 2]
```diff
@@ -26,9 +26,9 @@ include::partial$block-highlight.adoc[]
 .Context Clarification
 [CAUTION]
 --
-This content covers Inter-Sync Gateway Replication using the Inter-Sync Gateway Replication (v1) protocol.
+This content covers SG Replicate replications.
```
Comment (rajagp) [inline modules/ROOT/pages/_partials/block-caveats.adoc]:
```suggestion
This content covers  SG Replicate - inter-Sync Gateway replication in pre-2.8 versions of Sync Gateway.
```

File: modules/ROOT/pages/_partials/block-caveats.adoc [hunk 3]
```diff
@@ -26,9 +26,9 @@ include::partial$block-highlight.adoc[]
...
 // tag::icr-sgr2-redirect[]
-For documentation on later Inter-Sync Gateway Replication features see {xref-sgw-pg-icr-sgreplicate-sgr2}
+For documentation on Inter-Sync{nbsp}Gateway Replication features see {xref-sgw-pg-icr-sgreplicate}
```
Comment (rajagp) [inline modules/ROOT/pages/_partials/block-caveats.adoc]:
```suggestion
For documentation on inter-Sync{nbsp}Gateway Replication features introduced in Sync Gateway 2.8 see {xref-sgw-pg-icr-sgreplicate}
```

File: modules/ROOT/pages/_partials/block-caveats.adoc [hunk 4]
```diff
@@ -37,9 +37,9 @@ For documentation on later Inter-Sync Gateway Replication features see {xref-sgw
 .Context Clarification
 [CAUTION]
 --
-This content relates only to replications run using the latest version of Inter-Sync Gateway Replication (release 2.8+).
+This content relates only to Inter-Sync Gateway Replication.
```
Comment (rajagp) [inline modules/ROOT/pages/_partials/block-caveats.adoc]:
```suggestion
This content relates only to inter-Sync Gateway replication introduced in Sync Gateway 2.8.
```

File: modules/ROOT/pages/learn/icr-sgreplicate-sgr1.adoc [hunk 5]
```diff
@@ -1,15 +1,13 @@
-= Inter-Sync Gateway Replication v1.0
-:page-layout: article
+= SG Replicate
 :page-status: Deprecated
-:page-edition:
 :page-role: conceptual
-:description: Couchbase Inter-Sync Gateway Replication -- resilient, secure, scalable bidirectional synchronization of data between cloud and edge data centers.
+:description: Couchbase Mobile's SG Replicate protocol support inter-cluster replication
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate-sgr1.adoc]:
```suggestion
:description:  SG Replicate protocol supports inter-Sync Gateway replication
```

File: modules/ROOT/pages/learn/icr-sgreplicate-sgr1.adoc [hunk 6]
```diff
@@ -1,15 +1,13 @@
...
 
 include::partial$_std-hdr-sgw.adoc[]
 
 :topic-group: {tg-rep-icr}
 :param-related: {xref-sgw-pg-config-properties} | {xref-sgw-pg-rest-api-admin}
-:param-abstract: This content was deprecated in Sync Gateway v2.8. It provides an introduction to, and overview of, Couchbase Sync Gateway's Inter-Sync Gateway Replication v1 feature, which was replaced by v2 at that release
+:param-abstract: This content is deprecated. It provides an introduction to, and overview of SG Replicate, which was replaced by Inter-Sync{nbsp}Gateway Replication in release 2.8.
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate-sgr1.adoc]:
```suggestion
:param-abstract: This content is deprecated. It provides an introduction to, and overview of SG Replicate, which was replaced by a completely redesigned and rearchitected version in release 2.8.
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 7]
```diff
@@ -1,22 +1,19 @@
 = Inter-Sync Gateway Replication
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
= Inter-Sync Gateway Replication introduced in Sync Gateway 2.8
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 8]
```diff
@@ -1,22 +1,19 @@
...
-:page-layout: article
-:page-partial:
-:page-status:
-:page-edition:
 :page-role: conceptual
-:description: Couchbase Sync Gateway -- Inter-Sync Gateway Replication provides resilient, secure, scalable bidirectional synchronization of data between cloud and edge data centers.
+:description: Use Couchbase Sync Gateway's Inter-Sync Gateway Replication to keep clusters in different data centers in sync.
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
:description: Use inter-Sync Gateway replication to keep clusters in different mobile data centers in sync.
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 9]
```diff
@@ -1,22 +1,19 @@
...
 
 include::partial$_std-hdr-sgw.adoc[]
 
 :topic-group: {tg-rep-icr}
 :param-related: {xref-sgw-pg-config-properties} | {xref-sgw-pg-rest-api-admin} | {xref-sgw-pg-icr-running}
-:param-abstract: This content describes Couchbase Sync Gateway's Inter-Sync Gateway Replication feature (v2). It incorporates the content presented under Manage within the Nav bar to give an integrated view.
+:param-abstract: Inter-Sync Gateway Replication provides resilient, secure, scalable bidirectional synchronization of data between cloud and edge data centers. +
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
:param-abstract: inter-Sync Gateway replication supports resilient, secure, scalable bidirectional synchronization of data between cloud and edge data centers. +
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 10]
```diff
@@ -1,22 +1,19 @@
...
+It incorporates content presented under Manage>Replicate>Inter-Sync Gateway to give an integrated view.
 include::partial$block-abstract.adoc[]
 
 // Footnotes
-:fn-sgr1: footnote:sgr1[Inter-Sync Gateway Replication (v1) only]
-:fn-sgr2: footnote:sgr2[Inter-Sync Gateway Replication (v2) only]
-:fn-arch: footnote:arch[This architecture is also known as _ship-to-shore_ or _hub-and-spoke_.]
+:fn-sgr1: footnote:sgr1[SG{nbsp}Replicate only]
+:fn-sgr2: footnote:sgr2[Inter-Sync Gateway Replication only]
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]: I do not have context for this. Again, don't name this thing...refer to it as version of inter-Sync Gateway replication introduced in 2.8

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 11]
```diff
@@ -59,9 +56,13 @@ A typical architecture for this use cases is shown in <<icr-cloud-to-edge>>
 image::icr-cloud-to-edge200712.svg[,600]
 
 === Active-to-active Mobile Synchronization
-Here active mobile clusters at the edge are replicated between geographically separate cloud-based Sync Gateway deployments.
+Here edge clusters containing active Sync{nbsp}Gateway nodes are replicated between geographically separate cloud-based Sync Gateway deployments.
+An ideal use-case for Inter-Sync{nbsp}Gateway Replication, which was designed to keep clusters in different data centres in sync.
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
An ideal use-case for inter-Sync{nbsp}Gateway replication, which was designed to keep clusters in different data centers in sync.
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 12]
```diff
@@ -59,9 +56,13 @@ A typical architecture for this use cases is shown in <<icr-cloud-to-edge>>
...
 
-A typical architecture for this use cases is shown in <<icr-active-mobile>>
+The Couchbase Server cross data center replication API (xref:server:manage:manage-xdcr/xdcr-management-overview.adoc[XDCR]) is similarly used to replicate between Couchbase Server clusters.
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
The Couchbase Server Cross Data Center Replication API (xref:server:manage:manage-xdcr/xdcr-management-overview.adoc[XDCR]) is similarly used to replicate between Couchbase Server clusters.
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 13]
```diff
@@ -59,9 +56,13 @@ A typical architecture for this use cases is shown in <<icr-cloud-to-edge>>
...
+*However*, Inter-Sync{nbsp}Gateway Replication is specifically designed for Sync Gateway deployment and it *must* be used for replication between clusters containing active Sync{nbsp}Gateway nodes.
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
*However*, inter-Sync{nbsp}Gateway replication is specifically designed for Sync Gateway deployments and it *must* be used for replication between clusters containing active Sync{nbsp}Gateway nodes. 

Active Sync Gateway nodes are nodes with incoming writes; these may be from Couchbase Lite clients, the REST API, or through the Couchbase Server.
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 14]
```diff
@@ -73,7 +74,7 @@ image::icr-active-mobile-sync200713.svg[,600]
 _In this section_:  <<context>>  |  <<protocol>>  |  <<types-of-replication>>  |  <<delta-sync>>  |  <<directionality>>  |  <<security>>
 
 === Context
-Sync Gateway supports the ability to run replications between Sync Gateway clusters using Inter-Sync Gateway Replication (v2), with the {glos-term-active-replicator} {glos-term-synchronizing} changes between two {glos-term-sync-gateway-databases}.
+Sync Gateway supports the ability to run replications between Sync Gateway clusters using Inter-Sync Gateway Replication, with the {glos-term-active-replicator} {glos-term-synchronizing} changes between two {glos-term-sync-gateway-databases}.
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
Sync Gateway supports the ability to run replications between Sync Gateway clusters using a new websockets based protocol, with the {glos-term-active-replicator} {glos-term-synchronizing} changes between two {glos-term-sync-gateway-databases}.
```

File: modules/ROOT/pages/learn/icr-sgreplicate.adoc [hunk 15]
```diff
@@ -88,9 +89,9 @@ _Related configuration elements_:  {xref-sgw-pg-config-properties-databases}  |
 
 === Protocol
 
-Inter-Sync Gateway Replication (v2), are websocket-based.
+Replications run using Inter-Sync{nbsp}Gateway Replication, are websocket-based.
```
Comment (rajagp) [inline modules/ROOT/pages/learn/icr-sgreplicate.adoc]:
```suggestion
inter-Sync{nbsp}Gateway replications are based on websockets. This is the exact same protocol that is used for replication with Couchbase Lite 2.x clients.
```

**Review by rajagp — CHANGES_REQUESTED** [review]
I have made suggestions . But please do another pass to ensure that you are not treating "Inter-Sync Gateway replication" as a name ...in fact some of the original text was fine  ...

**Comment by ibsoln** [comment]
Suggestions applied. 

> I have made suggestions . But please do another pass to ensure that you are not treating "Inter-Sync Gateway replication" as a name ...in fact some of the original text was fine ...

@rajagp I will make any further changes as a new PR.

**Comment by ibsoln** [comment]
Suggestions all applied

**Pull request merged**